test/search/test_search.cpp

```diff
@@ -94,7 +94,7 @@ PointCloud<PointXYZ>::Ptr unorganized_sparse_cloud (new PointCloud<PointXYZ>);
 /** \brief unorganized point cloud*/
 PointCloud<PointXYZ>::Ptr unorganized_grid_cloud (new PointCloud<PointXYZ>);
 
-/** \brief uniform distributed random number generator for unsigned it in range [0;10]*/
+/** \brief uniform distributed random number generator for unsigned int in range [0;10]*/
```

Comment:
I hope this came on a separate commit. :sweat_smile:

Reply:
It's just this particular one, because when we squash your commits everything is gonna be bundled under the same thing, and this change appears out of nowhere and has nothing to do with the new fixed depth iterator. 

At the end we can still fix if we want, because it's a minor simple change.

Reply:
Ok, I rewrite the history so for git I never touch this line.